CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ enable_language(CXX)
 #####################################################################
 
 include(CheckCXXCompilerFlag)
+check_cxx_compiler_flag("-std=c++11" COMPILER_SUPPORTS_CXX11)
+if(COMPILER_SUPPORTS_CXX11)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=gnu++0x")
```

Comment:
Well you have to do a check with `check_cxx_compiler_flag()` as C++11 is only supported in >=gcc-4.8 and hence VOTCA won't compile with older compilers.